netutils/nng/Make.defs

```diff
@@ -21,9 +21,7 @@
 ifneq ($(CONFIG_NETUTILS_NNG),)
 CONFIGURED_APPS += $(APPDIR)/netutils/nng
 
-CFLAGS += ${shell $(INCDIR) $(INCDIROPT) "$(CC)" \
-	$(APPDIR)/netutils/nng/nng/include}
-CXXFLAGS += ${shell $(INCDIR) $(INCDIROPT) "$(CC)" \
-    $(APPDIR)/netutils/nng/nng/include}
+CFLAGS += ${INCDIR_PREFIX}$(APPDIR)/netutils/nng/nng/include
```

Comment:
move the change to the first patch

Reply:
It would be easier for me to just squash everything into one patch?

Reply:
Yes, could.